BWEnv/fbs/messages.fbs

```diff
@@ -107,3 +107,144 @@ table Message {
 }
 
 root_type Message;
+
+table Action {
+  action:[int];
+  uid:int;
+  aid:int;
+}
+
+table Resources {
+  ore:int;
+  gas:int;
+  used_psi:int;
+  total_psi:int;
+  upgrades:long;
+  upgrades_level:long;
+  techs:long;
+}
+
+struct Order {
+  first_frame:int; // First frame number where order appeared
+  type:int; //  Maps to BWAPI::Orders::Enum
+  targetId:int;
+  targetX:int;
+  targetY:int;
+  // Current implementation has equivalence operator that compares vars other than first_frame
```

Comment:
Might be cleared to say "that ignores first_frame". The same (ignoring frame) is true for UnitCommand.